ffv1.md

```diff
@@ -868,6 +891,78 @@ RFC:`log2_v_chroma_subsample` indicates the subsample factor, stored in powers t
 | 0     | extra `Plane` is not present |
 | 1     | extra `Plane` is present     |
 
+### plane_type
+
+`plane_type` indicates the semantic type of the plane.
+
+|value    | plane_type                                      |
+|---------|:------------------------------------------------|
+| 0       | Unspecified                                     |
+| 1       | Emmissive                                       |
+| 2       | Absorptive                                      |
+| 3       | Reflective                                      |
+| 4       | Retro-Reflective                                |
+| 5       | Mask                                            |
+| 6       | Depth                                           |
+| 7       | Motion                                          |
+| 8       | Error                                           |
+| 9       | Force Field                                     |
+| 10      | Polarization                                    |
+| 11-999  | Reserved                                        |
+
+### per_plane_metadata_count
+
+`per_plane_metadata_count` allows future extension of the information we store about planes
+
+### center_frequency
+
+`center_frequency` indicates the center frequency in hz. Or 0 if unknown or not applicable.
```

Comment:
nit `Hz`

Reply:
Sorry for jumping in (3 years+ later) :smile: :
Does any of you have any particular real-world examples in mind (or access to them)?
Like the xrays, long radio wave images, air/vacuum/etc recordings @michaelni mentioned?

I'm wondering if it'd make sense not to raw-parametrize this, but rather define/offer presets which can be tuned in the code, rather than as arguments? Then access these interpretation options by a given name shortcut?
(If I understood the subject alright)

I assume that any area of expertise that would create such recordings will very likely have some sets of "common parameters" - therefore domain-specific "insider terms" for those imaging types.

Like FFmpeg having "dvd_pal" containing a set of sane/valid parameters, we could have something like "xray_10nm" or "10nm-water-hot". Dunno. Something like that?


Wouldn't this also allow arbitrary choice of data-types (per imaging-type) for proper calculations?
Integers, fractions, etc - different strokes for different (imaging type) folks so to say :stuck_out_tongue_winking_eye: 


If I'm right, then this is tricky to handle in a Standards paper definition... But if x264 can be a standard and do presets, so could ffv1, I presume? :wink:

Reply:
If you need samples there should be lots of public stuff from astronomy and satellites. For xrays and MRT the word to search for is probably dicom image. One source for public domain samples would be any images you own the rights to, like from a medical exam. I didnt really try to find free proper raw sample images for these things previously ...